docs/_includes/search-scripts.html

```diff
@@ -0,0 +1,48 @@
+<script>
+    {% assign variations_count = 0 %}
+    // This is an inline script attached to the window so that we can utilize
+    // jekyll's liquid variables, such as site.pages.
+    window.searchStore = {
+        {% for page in site.pages %}
+        '{{ page.url | slugify }}': {
+            'title': '{{ page.title | xml_escape }}',
+            'description': {{ page.description | markdownify | strip_html | strip_newlines | jsonify }},
+            'usage': {{ page.usage | markdownify | strip_html | strip_newlines | jsonify }},
+            'accessibility': {{ page.usage | markdownify | strip_html | strip_newlines | jsonify }},
+            'url': '{{ page.url | xml_escape }}',
+
+            // Determine the maximum number of variations present overall.
+            {% if page.variations.size > variations_count %}
+                {% assign variations_count = page.variations | size  %}
+            {% endif %}
+
+            // Cycle through the variation fields of this page.
+            {% for variation in page.variations %}
```

Comment:
This appears to significantly increase the build time. We'll want to balance the cost/benefit of including the variations in the search, or see if we could cache the indexing somehow.